### PR TITLE
Provide branding guidelines

### LINKED ISSUE #1090: Provide branding guidelines
Provide branding guidelines .

### PR DESCRIPTION
**Pull request by tomkerkhove**
Signed-off-by: Tom Kerkhove <kerkhove.tom@gmail.com>

Provide branding guidelines

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [ ] Tests have been added
- [ ] A PR is opened to update the documentation on https://github.com/kedacore/keda-docs
- [ ] Changelog has been updated

Relates on #1080
Fixes #1090

### CODE REVIEW

File: BRANDING.md
Comment (turbaszek) [inline BRANDING.md:11]:
```suggestion
For online usage, we recommend using the shadow while for print it is recommended to use our simple approach.
```
Should we add links to those versions? This should increase accessibility

File: README.md
Comment (turbaszek) [inline README.md:73]:
```suggestion
We provide guidelines around branding of KEDA and related logos, learn more in our [guidelines](./BRANDING.md).
```

File: BRANDING.md
Comment (tomkerkhove) [reply BRANDING.md]: I've used a different approach PTAL

File: README.md
Comment (tomkerkhove) [reply README.md]: Done, thanks.

**Review by turbaszek — APPROVED** [review]
Looks good but I'm not sure why the logos are not visible here: https://github.com/tomkerkhove/keda/blob/branding/BRANDING.md

**Comment by tomkerkhove** [comment]
Do you mean here?
image

Because I see them 🤔

**Comment by turbaszek** [comment]
If you see them then ok, it can be Github issue 🤷 
Screenshot 2020-09-09 at 15 01 21

**Review by zroubalik — APPROVED** [review]
LGTM!

I got it rendered correctly here on the PR directly: https://github.com/kedacore/keda/blob/ec8a6ad8283a766b6b4794e57a97cd15ca786d21/BRANDING.md

**Comment by tomkerkhove** [comment]
Will wait for @caniszczyk for final sign-off, want to make sure it complies with CNCF standard.

**Review by ahmelsayed — APPROVED** [review]

**Comment by tomkerkhove** [comment]
I'm merging this but feel free to let me know if something is not ok @caniszczyk

**Pull request merged**
